site/_posts/2020-09-07-customer-story-scott-plunkett.md

```diff
@@ -0,0 +1,60 @@
+---
+layout: blog-post
+title: ! "Scott Plunkett's customer story"
```

Comment:
Customer Story: Contaim adopts FusionAuth to manage their auth lifecycle

Reply:
Great call!